test/pantry-test.js

```diff
@@ -0,0 +1,515 @@
+const chai = require('chai');
+const expect = chai.expect;
+import User from '../src/user.js';
+import Pantry from '../src/pantry.js';
+
+// const Ingredients = require('../src/ingredients.js');
+let userData
+let user1;
+let user1Pantry;
+let userData2
+let justin
+let justinPantry
+let recipeData
+let halfUser
+let halfUserData
+let halfUserPantry
+
+describe('Pantry', () => {
+  beforeEach(() => {
```

Comment:
We should put this all in a pantryTestData.js file and import it in so that this can be a little smaller.